lib/install/management/appliance.go

```diff
@@ -599,6 +599,7 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 				"CS_PATH=" + settings.ClusterPath,
 				"POOL_PATH=" + settings.ResourcePoolPath,
 				"DS_PATH=" + conf.ImageStores[0].Host,
+				"VC_KEEPALIVE=20s",
```

Comment:
This is the keep alive between the port layer and the vSphere environment. Unfortunately at the moment the only means of triggering re-authentication is tied to the keepalive interval.
Ideally we'd switch to something that would trigger after a single operation fails but keepalive was simplest approach when @dougm added this.